### PR TITLE
Add yast2-rdp autotest module in cmdline.

### PR DESCRIPTION
**Pull request by badboywj**
Add yast-rdp.pm.
yast-rdp.pm: this module will test yast rdp service in cmd mode, and just support SLE15+.

this is the verification run:
SLE15       : http://10.67.20.116/tests/212
SLE15SP1 : http://10.67.20.116/tests/215

### CODE REVIEW

**Comment by wjn740** [comment]
@badboywj , please squash commits to 1 commit.

File: tests/yast2_cmd/yast_rdp.pm [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+use warnings;
+use testapi;
+use utils;
+use version_utils 'is_sle';
+
+sub run {
+    select_console 'root-console';
+
+    # yast2-rdp was added after SLE12SP2.
+    if (is_sle('15+')) {
```
Comment (wjn740) [inline tests/yast2_cmd/yast_rdp.pm]: If this feature is added after SLE12SP2, than SLE12SP3 if is supported? If yes, this condition might need a branch for SLE12 product line.

Comment (badboywj) [reply tests/yast2_cmd/yast_rdp.pm]: I have corrected it. Thank you.

Comment (badboywj) [reply tests/yast2_cmd/yast_rdp.pm]: I have corrected it. Thank you.

File: tests/yast2_cmd/yast_rdp.pm [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+use version_utils 'is_sle';
+
+sub run {
+    select_console 'root-console';
+
+    # yast2-rdp was added after SLE15.
+    if (is_sle('15+')) {
+        zypper_call("in yast2-rdp", exitcode => [0, 102, 103]);
+    } else {
+        die "This version of product is NOT supported.";
```
Comment (dzedro) [inline tests/yast2_cmd/yast_rdp.pm]: If test should run only on 15+ then define it in main_common.pm, no need to fail on sle12

Comment (badboywj) [reply tests/yast2_cmd/yast_rdp.pm]: I see, your  point is good. And I have improve the code.

File: lib/main_common.pm [hunk 3]
```diff
@@ -1409,6 +1409,9 @@ sub load_extra_tests_y2uitest_gui {
 sub load_extra_tests_y2uitest_cmd {
     loadtest 'yast2_cmd/yast_lan';
     loadtest 'yast2_cmd/yast_timezone';
+    if (is_sle('15+')) {
+        loadtest 'yast2_cmd/yast_rdp';
+    }
```
Comment (dzedro) [inline lib/main_common.pm]: just hint, I guess people like more one liners ^^
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7765/files#diff-d3e0a6c7df772cff7741832a6250d5afL1405

Comment (badboywj) [reply lib/main_common.pm]: yes, it looks better.

**Pull request merged**